app/views/trial_feedbacks/show.html.slim

```diff
@@ -0,0 +1,39 @@
+h1= t_title(:show)
+
+.table-responsive.m-t-30.m-b-20
+  table.table.table-no-border-top
+    tbody
+      - if policy(TrialFeedback).superadmin_priviledges?
+        tr
+          td= t_model(Volunteer)
+          td= link_to @trial_feedback.volunteer.contact.full_name, volunteer_path(@trial_feedback.volunteer)
+      - if @trial_feedback.assignment?
+        tr
+          td= t_model(Assignment)
+          td= link_to @trial_feedback.feedbackable.to_label, assignment_path(@trial_feedback.feedbackable)
+        tr
+          td= t_model(Client)
```

Comment:
Is a Link to Client and Volunteer really needed, because a link to the Assignment could also do the job? Or not?
If not, it could be much slicker with:
```slim
tr
  td= t_model(@trial_feedback.feedbackable.class)
  td= link_to @trial_feedback.feedbackable.to_label, url_for(@trial_feedback.feedbackable)
```

Reply:
I think that's the consistent behaviour they are looking for to access the related persons profile. That's why I would leave it as it is.